src/OpenLoco/Environment.cpp

```diff
@@ -216,10 +216,17 @@ namespace OpenLoco::Environment
         auto& configLastSavePath = Config::getNew().last_save_path;
         if (!configLastSavePath.empty())
         {
-            auto directory = fs::u8path(configLastSavePath);
-            if (fs::is_directory(directory))
+            // Getting the directory can fail if config is bad.
```

Comment:
What's the recovery path? The catch block is empty.

Reply:
The recovery path is already set on L215 it defaults to path_id::save